wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -597,8 +597,11 @@ class KotlinGenerator private constructor(
   private fun getDefaultValue(field: Field): CodeBlock {
     val internalClass = ClassName("com.squareup.wire.internal", "Internal")
     return when {
-      isEnum(field) ->
+      isEnum(field) -> if (field.default != null) {
```

Comment:
@mc-lovin is it ever possible for the default value to not be null? If not - we can simplify this.

Reply:
yes, we can provide a default value via the proto 

```
  enum PhoneType {
    MOBILE = 0;
    HOME = 1;
    /** Could be phone or fax. */
    WORK = 2;
  }

  message PhoneNumber {
    // The customer's phone number.
    required string number = 1;
    // The type of phone stored here.
    optional PhoneType type = 2 [default = HOME];
  }
```